# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/grantlee/hotifx-qlist.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- grantlee-5.1.0/templates/lib/lexer_p.h.vanilla	2025-02-10 16:10:58.689448942 +0100
+++ grantlee-5.1.0/templates/lib/lexer_p.h	2025-02-10 16:12:25.059450602 +0100
@@ -23,6 +23,7 @@
 
 #include "textprocessingmachine_p.h"
 #include "token.h"
+#include <QList>
 
 template <typename T> class QList;
 
